v1.0/Obsolete/Part 2: File Format Policies.md

```diff
@@ -43,9 +43,9 @@ The following table describes the requirements for data quality when preserving
 | ODS_3 | Extension | The file **MUST** have extension “.fods”, “.ods” or “ots”. |
 | ODS_4 | External data | The file **MUST NOT** have any references to external data. This includes data connections, RealTimeData functions, cell formula references and objects. |
 | ODS_5 | Embedded objects | The file **MUST NOT** have any embedded objects, which are in violation of your organisation’s file format policies. |
-| ODS_6 | Cell values | The file **SHOULD** have cell values or objects. |
+| ODS_6 | Content | The file **SHOULD** have cell values or objects. |
```

Comment:
I suspect that we can do better with the text here, it's a little unclear and may need some explanation.

Reply:
Which wording do you suggest?

Reply:
Damn you and your trick questions ;)